data/autoyast_sle12/create_hdd/create_hdd_sles_regression_s390x.xml.ep

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I haven't found partition section, we'd better to set it by ourself while not only depend on autoyast to create it by default to avoid unexpected issue.

Reply:
I checked other *_s390x.xml file, they don't set partition. I don't get any problem at here.

Reply:
Without the partitioning section may meet unexpected result, the default behavior of autoyast is unreliable, please take this as an example that we need add partitioning section, https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17066

Reply:
Checked the log： /dev/disk/by-path/ccw-0.0.0000-part2 swap swap defaults 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 / btrfs defaults 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /boot/grub2/s390x-emu btrfs subvol=@/boot/grub2/s390x-emu 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /home btrfs subvol=@/home 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /opt btrfs subvol=@/opt 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /srv btrfs subvol=@/srv 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /tmp btrfs subvol=@/tmp 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /usr/local btrfs subvol=@/usr/local 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/cache btrfs subvol=@/var/cache 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/crash btrfs subvol=@/var/crash 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/lib/libvirt/images btrfs subvol=@/var/lib/libvirt/images 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/lib/machines btrfs subvol=@/var/lib/machines 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/lib/mailman btrfs subvol=@/var/lib/mailman 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/lib/mariadb btrfs subvol=@/var/lib/mariadb 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/lib/mysql btrfs subvol=@/var/lib/mysql 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/lib/named btrfs subvol=@/var/lib/named 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/lib/pgsql btrfs subvol=@/var/lib/pgsql 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/log btrfs subvol=@/var/log 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/opt btrfs subvol=@/var/opt 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/spool btrfs subvol=@/var/spool 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /var/tmp btrfs subvol=@/var/tmp 0 0
UUID=50c61041-a4b0-4f7a-9a98-59e61af44f43 /.snapshots btrfs subvol=@/.snapshots 0 0
/dev/disk/by-path/ccw-0.0.0000-part1 /boot/zipl           ext2       acl,user_xattr        1 2
This one can match my requirement. Don't need configure partitions for zkvm.

Reply:
We can check later about this, but still one thing left to do for partitioning: to get more space we'd create hdd without /home while if you do not set partitioning section it will be created by default that's not what we expected. Anyway, we need one more PR to fix this.